modules/swagger-codegen/src/main/resources/python/asyncio/setup.mustache

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+
+{{>partial_header}}
+
+import sys
+from setuptools import setup, find_packages
+
+NAME = "{{{projectName}}}"
+VERSION = "{{packageVersion}}"
+{{#apiInfo}}
+{{#apis}}
+{{^hasMore}}
+# To install the library, run the following
+#
+# python setup.py install
+#
+# prerequisite: setuptools
+# http://pypi.python.org/pypi/setuptools
+
+REQUIRES = ["aiohttp", "urllib3", "six >= 1.10", "certifi", "python-dateutil"]
```

Comment:
aside from the rest.py, the other changes were fairly minor to the files themselves. I'm happy to switch this over to a template conditional for the other cases.